docs/orleans/benefits.md

```diff
@@ -1,75 +1,76 @@
+The Orleans programming model raises productivity, regardless of expertise level, by providing the following key abstractions, guarantees, and system services.
 
 ### Familiar object-oriented programming (OOP) paradigm
 
-Grains are .NET classes that implement declared .NET grain interfaces with asynchronous methods. Grains appear to the programmer as remote objects whose methods can be directly invoked. This provides the programmer the familiar OOP paradigm by turning method calls into messages, routing them to the right endpoints, invoking the target grain's methods, and dealing with failures and corner cases transparently.
+Grains are .NET classes implementing declared .NET grain interfaces with asynchronous methods. Grains appear as remote objects whose methods can be directly invoked. This provides the familiar OOP paradigm by turning method calls into messages, routing them to the right endpoints, invoking the target grain's methods, and transparently handling failures and corner cases.
 
 ### Single-threaded execution of grains
 
-The runtime guarantees that a grain never executes on more than one thread at a time. Combined with the isolation from other grains, the programmer never faces concurrency at the grain level, and never needs to use locks or other synchronization mechanisms to control access to shared data. This feature alone makes the development of distributed applications tractable for non-expert programmers.
+The runtime guarantees a grain never executes on more than one thread at a time. Combined with isolation from other grains, developers never face concurrency at the grain level and never need locks or other synchronization mechanisms to control access to shared data. This feature alone makes developing distributed applications tractable, even for non-expert programmers.
 
 ### Transparent activation
 
-The runtime activates a grain only when there is a message for it to process. This cleanly separates the notion of creating a reference to a grain, which is visible to and controlled by application code, and physical activation of the grain in memory, which is transparent to the application. This is similar to virtual memory in that it decides when to "page out" (deactivate) or "page in" (activate) a grain; The application has uninterrupted access to the full "memory space" of logically created grains, whether or not they are in the physical memory at any particular point in time.
+The runtime activates a grain only when there's a message for it to process. This cleanly separates creating a grain reference (controlled by application code) and physical activation of the grain in memory (transparent to the application). This is similar to virtual memory deciding when to "page out" (deactivate) or "page in" (activate) a grain. The application has uninterrupted access to the full "memory space" of logically created grains, whether they are in physical memory at any given time.
```

Comment:
Similarly, I think the use of "deciding" here actually changes the meaning of the sentence. It's not virtual memory deciding to page out a grain right?